server/src/main/resources/rules/rules.js

```diff
@@ -1,4 +1,4 @@
-// Version: 5.31
+// Version: 5.32
```

Comment:
Not a review, but as a general note: there is currently another PR with rules changes (https://github.com/candlepin/candlepin/pull/2223), which also increases the version to 5.32, so whichever of these gets merged first, the other will have to make sure to update it to 5.33. Similarly if this PR also gets patched into the candlepin-2.3-HOTFIX branch, as there is a PR for that branch too (https://github.com/candlepin/candlepin/pull/2224).